### PR TITLE
Fix missing @export_flags_2d_navigation/@export_flags_3d_navigation in annotation list

### PR DESCRIPTION
**Pull request by Cretezy**
Listed on https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_exports.html but missing in GDScripts basics list

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -347,51 +347,55 @@ Both of these are the same::
...
+| ``@export_flags_2d_navigation`` |                                                                                                   |
+|                                 |                                                                                                   |
+| ``@export_flags_3d_render``     |                                                                                                   |
+|                                 |                                                                                                   |
+| ``@export_flags_3d_physics``    |                                                                                                   |
+|                                 |                                                                                                   |
+| ``@export_flags_3d_navigation`` |                                                                                                   |
 +------------------------------+---------------------------------------------------------------------------------------------------+
```
Comment (YuriSizov) [inline tutorials/scripting/gdscript/gdscript_basics.rst:399]: Table formatting needs to be aligned.

**Comment by YuriSizov** [comment]
I think we may want to remove that table and link to the class reference instead. It duplicates the same information already present there, and, for export annotations, in the linked dedicated article.

**Comment by Cretezy** [comment]
@YuriSizov I agree, that would be good too. I do see the value of having a list of the symbols there, especially since it's a reference page, where someone can search any of the annotations.

**Comment by skyace65** [comment]
I agree with Yuri on this, listing out every export annotation is excessive. I don't think taking it out makes it harder to find information, if they're looking for a specific export annotation the docs search would bring up the export page. And if they're searching just the reference page to learn what annotations there are, they would find that section which links to the export page.

I think replacing the table with a bullet point list and links is the way to go. Just note on the link for exports that it includes information on other export annotations such as `export_enum` or something along those lines.

**Comment by mhilbrunner** [comment]
Also in favor of not duplicating information :)

**Comment by YuriSizov** [comment]
I'll see what we can do about this, but in the meantime it seems that there is a bigger PR that also covers this enhancement, so I'll close this one as superseded by https://github.com/godotengine/godot-docs/pull/6481.

Thanks for your contribution nonetheless!

**Pull request closed without merging**